cmd/operator/main.go

```diff
@@ -239,6 +239,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Update WICD RBAC with proper namespace
+	wicdResourceName := "windows-instance-config-daemon"
+	if err := configMapReconciler.UpdateWICDRoleBindingNamespace(wicdResourceName); err != nil {
```

Comment:
I suggest not updating this always but rather only do this if a non-default namespace is being used.
```golang
if watchNamespace != controller.WMCONamespace {
    // Update WICD RoleBinding
}
```